chunky/src/java/se/llbit/chunky/world/biome/MapBiomePalette.java

```diff
@@ -0,0 +1,50 @@
+package se.llbit.chunky.world.biome;
+
+import it.unimi.dsi.fastutil.objects.Reference2IntMap;
+
+import it.unimi.dsi.fastutil.objects.Reference2IntOpenHashMap;
+import se.llbit.util.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/*
+* currently unused as no world has enough biomes for this to even be relevant
+* would probably need to be >10000 biomes to even consider it
+*/
```

Comment:
Is it used as a fallback if the other palette _does_ grow too big or is this completely unused? Should we remove it then?

Reply:
completely unused, I will remove it, if we need it I can just look at this PR again